src/loginServices.js

```diff
@@ -110,15 +115,50 @@ export function getBindings(okapiUrl, store, tenant) {
     });
 }
 
+function getErrorMessage(error = {}) {
+  const { code = 'default.error', parameters } = error;
+  const values = parameters ?
+    parameters.reduce(
+      (res, key) => {
+        res[key.key] = key.value;
```

Comment:
res[key.key] looks pretty strange. Is key an object? If yes maybe it should be renamed and you also can use destructuring? Btw, use short notation. See comment below.